.github/workflows/e2e-test.yml

```
@@ -0,0 +1,49 @@
+name: E2E Test
+on:
+  pull_request:
+    paths:
+      - .github/workflows/e2e-test.yml
```

Let's rename the file and change this line
```suggestion
      - .github/workflows/web-e2e-test.yml
```